test/unit/run.cpp

```diff
@@ -63,6 +96,7 @@ int main (int argc, char* const argv[])
 
     if (result == 0)
     {
+        auto tmp_base_dir = catch_temporary_path::create_dir("");
         result = session.run();
```

Comment:
image_io_test. I don't want a hundred files directly in /tmp, so they go in this subdirectory (/tmp/mapnik-test-unit)